### ISSUE TITLE
Uncaught ReferenceError: positionMenu is not defined

### ISSUE THREAD

**Issue by aditiapratama** [suda/tool-bar#184]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.16.0-dev-2e72f39 x64
**Electron**: 1.3.13
**OS**: "Fedora release
**Thrown From**: [tool-bar](https://github.com/suda/tool-bar) package 1.0.1

### Stack Trace

Uncaught ReferenceError: positionMenu is not defined

```
At /home/aditia/.atom/packages/tool-bar/lib/tool-bar-view.js:197

ReferenceError: positionMenu is not defined
    at ToolBarView.updateMenu (tool-bar-view.js:197:26)
    at ToolBarView.updatePosition (tool-bar-view.js:183:10)
    at ToolBarView.show (tool-bar-view.js:234:10)
    at ToolBarView.subscriptions.add.atom.config.onDidChange (tool-bar-view.js:41:16)
    at /app.asar/src/config.js:635:20
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at Config.module.exports.Config.emitChangeEvent (/app.asar/src/config.js:826:29)
    at Config.module.exports.Config.setRawValue (/app.asar/src/config.js:612:19)
    at Config.module.exports.Config.set (/app.asar/src/config.js:227:14)
    at SettingsPanel.module.exports.SettingsPanel.set (/app.asar/node_modules/settings-view/lib/settings-panel.js:201:28)
    at /app.asar/node_modules/settings-view/lib/settings-panel.js:215:26)
    at HTMLSelectElement.dispatch (/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLSelectElement.elemData.handle (/app.asar/node_modules/jquery/dist/jquery.js:4121:28)
```

### Commands

```
     -4:33.1.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always.theme-base16-syntax.theme-nucleus-dark-ui)
     -2:04.0 settings-view:open (input.hidden-input)
     -0:53.7.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.5 
aligner-php 1.0.0 
an-old-hope-syntax 1.0.2 
atom-alignment 0.12.1 
atom-beautify 0.29.17 
atom-bootstrap3 1.2.12 
atom-css-unit-converter 1.1.1 
atom-dark-fusion-syntax 2.0.3 
atom-html-preview 0.1.22 
atom-live-server 2.1.1 
atom-material-syntax 1.0.2 
atom-material-syntax-dark 0.2.7 
atom-material-syntax-light 0.4.6 
atom-material-ui 1.3.9 
atom-monokai 0.10.9 
atom-typescript 10.1.14 
autocomplete-emojis 2.5.0 
autocomplete-paths 1.0.5 
autocomplete-php 0.3.7 
autocomplete-python 1.8.57 
autoprefixer 3.6.1 
base16-syntax 1.8.0 
batman-syntax 1.1.0 
cake-syntax 1.0.2 
color-picker 2.2.5 
duotone-dark-space-syntax 2.1.0 
duotone-dark-syntax 2.1.0 
duotone-light-syntax 2.1.0 
eclectic-syntax 0.2.1 
emmet 2.4.3 
ex-mode 0.13.1 
expose 0.14.0 
file-icons 2.0.15 
firewatch-syntax 1.1.1 
fizzy 0.21.0 
font-viewer 0.3.1 
fonts 2.0.0 
genesis-ui 0.5.0 
git-blame 0.4.11 
git-control 0.8.2 
git-diff-details 1.4.0 
git-history 3.2.0 
git-log 0.4.1 
git-projects 1.17.0 
git-time-machine 1.5.6 
graphite-ui 0.15.1 
gruvbox-plus-syntax 1.3.1 
gruvbox-syntax 2.4.1 
hey-pane 0.2.0 
highlight-selected 0.12.0 
isotope-ui 2.7.1 
linter 1.11.23 
linter-bootlint 1.0.4 
linter-csslint 1.3.4 
linter-jscs 4.1.1 
linter-php 1.3.1 
linter-pycodestyle 2.0.0 
linter-rubocop 0.5.3 
linter-stylint 2.2.6 
merge-conflicts 1.4.4 
minimap 4.26.8 
minimap-autohide 0.10.1 
monokai 0.22.0 
monokai-seti 0.7.0 
multi-cursor 2.1.5 
native-ui 0.20.4 
neon-selection 0.3.2 
neutron-ui 0.4.0 
nord-atom-syntax 0.6.0 
nord-atom-ui 0.10.1 
nucleus-dark-ui 0.11.0 
numix-ui 0.2.2 
octocat-syntax 1.3.1 
open-in-browser 0.4.7 
open-recent 5.0.0 
open-terminal-here 2.3.0 
pigments 0.39.0 
platformio-ide-terminal 2.3.1 
predawn-syntax 1.0.4 
predawn-ui 1.0.7 
pretty-json 1.6.1 
project-manager 3.3.3 
project-plus 0.9.0 
python-tools 0.6.9 
Remote-FTP 0.10.7 
script 3.14.1 
sort-lines 0.14.0 
spacegray-dark-ui 0.12.0 
split-diff 1.2.0 
state-of-nature-syntax 0.1.0 
Sublime-Style-Column-Selection 1.7.3 
svg-preview 0.11.0 
symbols-tree-view 0.13.2 
sync-settings 0.8.1 
tag 0.4.0 
tango-syntax 0.4.1 
tool-bar 1.0.1 
tool-bar-almighty 0.8.1 
tree-view-git-status 1.3.1 
trendy-dark-syntax 0.2.0 
trendy-light-ui 0.28.1 
vim-mode-plus 0.82.3 
w3c-validation 0.4.0 
wordpress-api 1.2.2 
yeti-syntax 0.1.2 
yeti-ui 0.1.6 
zenburn 3.1.0 
zip-folder 3.2.0 
```

**Comment by harindu95** [comment]
I have the same issue. It seems there is a typo "positionMenu" instead of "positionsMenu" in line 197: lib/tool-bar-view.js

**Comment by suda** [comment]
This has been fixed in 539bb870c1ed2b272f05b7ffc0bf862ace48fb44
